src/app/styles/vendors/_slick-theme.scss

```diff
@@ -0,0 +1,194 @@
+@charset "UTF-8";
```

Comment:
Is this the same as the slick theme included in the bower package? If so, it should be picked up automatically by the gulp build process.